ui/public/src/install-ses-lockdown.js

```diff
@@ -7,7 +7,7 @@ import '@endo/eventual-send/shim'; // adds support needed by E
 // can still access powerful globals, but this start compartment can use `new Compartment(...)`
 // to evaluate code with stricter confinement.
 lockdown({
-  errorTaming: 'unsafe',
+  errorTaming: 'unsafe', // TODO Wen safe?
```

Comment:
I think [process.env.NODE_ENV](https://github.com/parcel-bundler/parcel/issues/1295#issuecomment-386707292) should work here as well.